esmvaltool/cmor/fix.py

```diff
@@ -1,18 +1,84 @@
-"""CMOR fixer for Iris cubes."""
+"""
+Apply automatic fixes for known errors in cmorized data
+
+All functions in this module will work even if no fixes are available
+for the given dataset. Therefore is recommended to apply them to all
+variables to be sure that all known errors are
+fixed.
+
+"""
 from ._fixes.fix import Fix
 from .check import _get_cmor_checker
 
 
 def fix_file(filename, short_name, project, model, output_dir):
-    """Fix errors that prevent loading or can not be fixed in the cube."""
+    """
+    Fix files before ESMValTool can load them
+
+    This fixes are only for issues that prevent iris to load the files or
```

Comment:
These fixes are only for issues that prevent iris from loading the cube or
that cannot be fixed after the cube is loaded.